docs/reference/commandline/dockerd.md

```diff
@@ -687,7 +685,7 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
-### Allow push of nondistributable artifacts
+### Allow push of non-distributable artifacts
```

Comment:
Remind me; we need to check the status on this one. I know the "foreign layers" feature  was deprecated by the OCI, and we may be enabling this by default now (or are planning to)